ui-particles-angular/projects/ui-particles-angular/src/lib/gio-form-cron/README.stories.mdx

```diff
@@ -5,6 +5,10 @@ import { Meta, Story } from '@storybook/addon-docs';
 # Form Cron
 
 Components to create a cron expression. Usable in angular forms.
+
+Cron expression is a string of 6 parts separated by space. Each part is a time definition. 
+This component are made for Spring cron expression only.
```

Comment:
What about putting a link to Spring Doc (https://spring.io/blog/2020/11/10/new-in-spring-5-3-improved-cron-expressions or https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/scheduling/support/CronExpression.html)

Reply:
yep